editor/filesystem_dock.cpp

```diff
@@ -1738,6 +1757,10 @@ FileSystemDock::FileSystemDock(EditorNode *p_editor) {
 	editor = p_editor;
 	path = "res://";
 
+	ED_SHORTCUT("filesystem_dock/copy_node_path", TTR("Copy Node Path"), KEY_MASK_CMD | KEY_C);
+	ED_SHORTCUT("filesystem_dock/duplicate", TTR("Duplicate"), KEY_MASK_CMD | KEY_D);
```

Comment:
I believe you should use `TTR("Copy Path")` and `TTR("Duplicate..")` here, as seen in:
- https://github.com/godotengine/godot/blob/34c988cfa92f19c232b65990704816ba1c7d2622/editor/filesystem_dock.cpp#L1603
- https://github.com/godotengine/godot/blob/34c988cfa92f19c232b65990704816ba1c7d2622/editor/filesystem_dock.cpp#L1605